resources/util_test.go

```diff
@@ -236,3 +238,79 @@ func TestLowerStructFieldNameToFieldName1(t *testing.T) {
 		return
 	}
 }
+
+func TestUnknownGroup(t *testing.T) {
+	gid, err := GetGID("unknowngroup")
+	if err == nil {
+		t.Errorf("expected failure, but passed with: %d", gid)
+	}
+}
+
+func TestUnknownUser(t *testing.T) {
+	uid, err := GetUID("unknownuser")
+	if err == nil {
+		t.Errorf("expected failure, but passed with: %d", uid)
+	}
+}
+
+func TestCurrentUserGroupByName(t *testing.T) {
+	// get current user
+	userObj, err := user.Current()
+	if err != nil {
+		t.Errorf("error trying to lookup current user: %s", err.Error())
+	}
+
+	currentUID := userObj.Uid
+	currentGID := userObj.Gid
+
+	var uid int
+	var gid int
+
+	// now try to get the uid/gid via our API (via username and group name)
+	if uid, err = GetUID(userObj.Username); err != nil {
+		t.Errorf("error trying to lookup current user UID: %s", err.Error())
+	}
+
+	if strconv.Itoa(uid) != currentUID {
```

Comment:
You can compare directly to userObj.Uid here. No need to store it in a variable. Same thing for the corresponding GID check both here and in TestCurrentUserGroupById.